### PR TITLE
S1t5

### PR DESCRIPTION
**Pull request by Parsley-Afroman**

### CODE REVIEW

File: db.php [hunk 1]
```diff
@@ -11,52 +11,53 @@ function getItems(object $db): array {
 function arraySplit(array $items) : string{
```
Comment (gabrielrowan) [inline db.php:11]: Charlie might ask you to rename this file from db.php to functions.php

Comment (gabrielrowan) [reply db.php:11]: Also I would add docblocks for your functions

File: db.php [hunk 2]
```diff
@@ -11,52 +11,53 @@ function getItems(object $db): array {
...
     $result = '';
     if (!count($items)) {
-        $result .= '<div>' . 'there aren\'t any database items to present' . '<div>';
+        $result .= '<div class = "empty">' . 'there aren\'t any database items to present' . '</div>';
```
Comment (gabrielrowan) [inline db.php:14]: You don't actually need to nest your foreach loop inside of an else statement here. You can just have your first if statement and then start the foreach loop on the next line

**Review by gabrielrowan — CHANGES_REQUESTED** [review]

File: db.php (hunk 1 above)
Comment (Parsley-Afroman) [reply db.php:11]: I have added doc blocks to my functions

File: db.php (hunk 2 above)
Comment (Parsley-Afroman) [reply db.php:14]: I have done this

File: index.php [hunk 3]
```diff
@@ -8,6 +8,7 @@
 <html lang="en">
 <header>
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply index.php:9]: lol, very true; I have changed it.

File: index.php [hunk 4]
```diff
@@ -8,6 +8,7 @@
...
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width">
     <link href="normalize.css" type="text/css" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply index.php:12]: I think I got confused as meta ones don't; I have now replaced this

File: index.php [hunk 5]
```diff
@@ -8,6 +8,7 @@
...
     <link href="style.css" type="text/css" rel="stylesheet">
     <title>Sami's Collection</title>
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply index.php:14]: Good advice, an oversight on my part that I have now included

File: style.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    .item {
+        width: 45vw;
+    }
+}@media only screen and (max-width: 750px ){
+    .item {
+        height: 100vh;
+        width: 100vw;
+    }
+}
+
```
[the remark this answers is not in the document]
Comment (Parsley-Afroman) [reply style.css:39]: no longer there :)

**Comment by gabrielrowan** [comment]
Perfect!! All good to merge

**Review by gabrielrowan — APPROVED** [review]
Fantastic :) All good and approved

**Pull request merged**